src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -198,18 +183,13 @@ public class TestConnectionCommand : PSCmdlet
         /// </summary>
         [Parameter(ParameterSetName = SourceParameterSet)]
         [Parameter(ParameterSetName = RegularParameterSet)]
-        [ValidateRange(int.MinValue, (int)1000)]
+        [ValidateRange((int)1, (int)1000)]
```

Comment:
The current [documentation](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/restart-computer?view=powershell-5.1) says using 0 == 32.  I think you should support that to avoid a breaking change.  We can probably safely not support negative values.

Reply:
yah - specifying a throttle of 0 which means 32 doesn't make any sense to me at all. The default is 32, which is fine (FWIW, throttling doesn't seem to be actually implemented). We have the opportunity to fix this somewhat strange behavior, by specifying a range which is actually useful.

Reply:
I'm ok with updating the documentation and making this breaking change